TamoStudy/src/model/language/EnglishLanguage.java

```diff
@@ -185,7 +185,7 @@ public EnglishLanguage() {
 		sofaText = "Sofa";
 		sunriseText = "Sunrise";
 		nightOutText = "Night Out";
-		enigmaText = "Enigma";
+		enigmaText = "Misterio";
```

Comment:
Remove

Reply:
remove..? Mistero? you're removing the engimaText entirely?

Reply:
No, the file that you edited was `EnglishLanguage.java` in this change